lte/gateway/fabfile.py

```diff
@@ -414,6 +415,25 @@ def bazel_integ_test_post_build(
         env.hosts = [gateway_host]
 
 
+def _setup_vm(host, name, ansible_role, ansible_file, destroy_vm, provision_vm):
```

Comment:
Would be nice to have type annotations for these two new functions.

Reply:
Normally I agree, but I'm not sure that they work well with fab. So the fabfiles are a bit of a special case.

Reply:
I get what you mean, makes sense.